frontend/src/modules/member/components/view/member-view-aside.vue

```diff
@@ -22,16 +22,31 @@
               ? 'hover:bg-gray-50 transition-colors cursor-pointer'
               : ''
           "
-          :href="
-            platform === 'hackernews'
-              ? `https://news.ycombinator.com/user?id=${socialIdentities.hackernews}`
-              : member.attributes.url?.[platform]
-          "
+          :href="identityUrl(socialIdentities, platform)"
```

Comment:
Shouldn't you just pass `platform`? And remove socialIdentities?
At least from the `identityUrl` it seems that it only needs the platform value

Reply:
Thanks for flagging this, I forgot to remove the `socialIdentities` parameter, will update!